source/getting_started/aiot_intelligent_edge/coco_for_devices/linux_coco_for_devices/quick_start_guide_linux_coco_for_devices.rst

```diff
@@ -3,6 +3,153 @@
 Quick Start Guide
 =================
 
```

Comment:
Add the introductory text back here. I didn't intend for it to be removed: "Build and run your first Device App in a few short steps by following the guide below."

Reply:
I didn't remove the "Build and run your first Device App in a few short steps by following the guide below"